libs/frontend/ui-build/src/components/component-menu/SizeInput.vue

```diff
@@ -59,6 +61,13 @@ const emit = defineEmits<{
   (e: 'update', size: string | undefined): void
 }>()
 
+// Set tabindex to -1 when right menu is not focused to make size input not tab-focusable when
+// right menu is not focused. In the builder, we can select the next component by pressing Tab
+// continuously. However, if the next component is currently not in the viewport, after the
+// scrollToComponent operation, the subsequent Tab press will focus on the first size input
+// instead of selecting the next component.
+const tabIndex = computed(() => (runtimeContext.rightMenuFocused.value ? 1 : -1))
```

Comment:
There's a strange behavior I don't understand; if the next component is in the viewport, pressing Tab would correctly select & gain focus on the next component. But if the next component is not in the viewport, after the scrollToComponent operation, the subsequent Tab press will focus on the first size input (probably because inputs are tab-focusable?) instead of selecting the next component.

https://github.com/samatechtw/pubstudio-builder/assets/59676941/a25709b5-d1a8-4657-a7d0-a362c9d028fa

Unfortunately, I didn't a clean way to solve this problem 😕. My current workaround is to only enable the tab navigation of size inputs when right menu is focused.

Reply:
I see! I guess that's just how the browser handles the tab key normally. Your fix seems to work well 🙏